index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Kalolo's Portfolio</title>
+    
+    <link rel="stylesheet" href="styles.css">
+
+</head>
+<body>
+    <nav class="nav-bar">
+
+    <div class="logo">
+        <a href="#">My logo</a>
+    </div>
+        <div class="menu-icon"> <img src="images/menu.png" alt="menu navbar"></div>
+    </nav>
```

Comment:
- [ ] The container of the section is a header tag, let us wrap our `nav tag` inside `header tag`.
- [ ] Nice to have logo on the left side, however let us add `text-decoration:none` to remove underline.